packages/web-react/src/components/Modal/README.md

```diff
@@ -308,10 +309,27 @@ const handleClose = () => setOpen(false);
 </Modal>
 ```
 
+## Feature Flag: Uniform Appearance on All Breakpoints
+
+The uniform appearance of modal dialog on all breakpoints is disabled by default. To enable it, either set the
+`$modal-enable-uniform-dialog` feature flag to `true` or use the `spirit-modal-enable-uniform-dialog` CSS class on any
```

Comment:
I think we missed somewhere in this paragraph that the feature flag is in SCSS, but it is probably mentioned in the readme.

Reply:
I added the word `Sass` to all relevant READMEs.